errs/error.go

```diff
@@ -188,63 +215,62 @@ func StatusCodeError(code int, e error, opts ...Option) error {
 	}
 }
 
-var seeLogs = "Please see the certificate authority logs for more info."
+var (
+	seeLogs = "Please see the certificate authority logs for more info."
+	// BadRequestDefaultMsg 400 default msg
+	BadRequestDefaultMsg = "The request could not be completed due to being poorly formatted or missing critical data. " + seeLogs
+	// UnauthorizedDefaultMsg 401 default msg
+	UnauthorizedDefaultMsg = "The request lacked necessary authorization to be completed. " + seeLogs
+	// ForbiddenDefaultMsg 403 default msg
+	ForbiddenDefaultMsg = "The request was forbidden by the certificate authority. " + seeLogs
+	// NotFoundDefaultMsg 404 default msg
+	NotFoundDefaultMsg = "The requested resource could not be found. " + seeLogs
+	// InternalServerErrorDefaultMsg 500 default msg
+	InternalServerErrorDefaultMsg = "The certificate authority encountered an Internal Server Error. " + seeLogs
+	// NotImplementedDefaultMsg 501 default msg
+	NotImplementedDefaultMsg = "The requested method is not implemented by the certificate authority. " + seeLogs
+)
 
 // InternalServerError returns a 500 error with the given error.
 func InternalServerError(err error, opts ...Option) error {
-	if len(opts) == 0 {
```

Comment:
Now it looks like we use more `errs.MyStatus(errors.New("foo bar zar")` than `errs.MyStatus(err)` we should consider what I proposed later:
```go
func errs.MyStatus(format string, args ...interface{}) error // here we can also apply options 
func errs.MyStatusErr(err, opts ...Option) error
```

Reply:
To keep having the stack trace the format method can do that using `errors.Errorf(format, argsWithoutOptions...)`